src/main/java/com/tabgok/heimdallr/HeimdallrServer.java

```diff
@@ -0,0 +1,20 @@
+package com.tabgok.heimdallr;
+
+import com.tabgok.entity.factory.MachineFactory;
+import com.tabgok.server.Gateway;
+
+/**
+ * Starts up an instance of the Heimdallr system, which consists of two parts:
```

Comment:
"Main class which starts the Heimdallr system. Startup consists of two parts:"

(note: probably not necessary to describe how it starts up, though)